lib/realtime/tenants/connect.ex

```diff
@@ -78,6 +87,21 @@ defmodule Realtime.Tenants.Connect do
   end
 
   @impl GenServer
+  def handle_continue(:setup_connected_users, state) do
+    %{check_connected_user_interval: check_connected_user_interval} = state
+    Process.send_after(self(), :check_connected_users, check_connected_user_interval)
+    {:noreply, state}
+  end
+
+  @impl GenServer
+  def handle_info(:check_connected_users, %{tenant_id: tenant_id} = state) do
+    if UsersCounter.tenant_users(tenant_id) > 0 do
```

Comment:
Here.

Reply:
And also only if it's been zero for greater than one minute.

Reply:
that is done since we're using `send_after` instead of `send` so the first check will only happen after `check_connected_user_interval`

Reply:
will add a test to reflect said use case

Reply:
changed test:
```

    test "if no users are connected to a tenant channel, stop the connection", %{
      tenant: %{external_id: tenant_id}
    } do
      Connect.lookup_or_start_connection(tenant_id, check_connected_user_interval: 500)
      assert {_, %{conn: _}} = :syn.lookup(Connect, tenant_id)

      # Not enough time has passed, connection still alive
      :timer.sleep(100)
      assert {_, %{conn: _}} = :syn.lookup(Connect, tenant_id)

      # Enough time has passed, connection stopped
      :timer.sleep(500)
      assert :undefined = :syn.lookup(Connect, tenant_id)
    end
```